### ISSUE TITLE
It would be nice to be able to join without having to create a Google account

### ISSUE THREAD

**Issue by turekg** [rse-aunz/rse-au#88]
Thanx and cheers!
Gaby

**Comment by manodeep** [comment]
Not really sure how to properly solve the underlying issue but if you share your email address, then I am happy to send you an invite to join.

(I do know that folks have signed up with their institutional emails but perhaps they were still signed into google with their personal accounts. Or perhaps the security settings have changed)

**Comment by mpfl** [comment]
Try emailing `rse-nz-au+subscribe@googlegroups.com` from the account you wish to subscribe with.

**Comment by turekg** [comment]
No, unfortunately it does not work, as it requests a password, which to me implies that it is expecting a google account. If you could manually add my email address ***@***.*** ***@***.***> maybe that will work
Cheers
Gaby

**Comment by turekg** [comment]
Apologies, it was the settings in my email client that asked for a password. I was able to send the email. All’s good
Cheers!
Gaby

**Comment by manodeep** [comment]
@turekg Ohh good! I see a pending request from tutanota.com - is that your email? If so, I will approve that join request

**Comment by turekg** [comment]
Yes it is me…
Cheers!
Gaby

**Comment by manodeep** [comment]
Done!

**Comment by PeiyuOwO** [comment]
Thanks!

**Comment by manodeep** [comment]
@PeiyuOwO I assume you would like to join the RSE-AUNZ mailing list - will you please confirm?

**Comment by PeiyuOwO** [comment]
yes, my supervisor introduced me this, but I couldn’t join via the invitation link. I confirm that I would like to join it, thanks!

**Comment by manodeep** [comment]
Invite sent

**Comment by PeiyuOwO** [comment]
Thanks! Have a nice day

**Comment by RichardLitt** [comment]
I believe this can be closed.
